src/components/HomePage/Banner.js

```diff
@@ -151,6 +153,23 @@ function renderInfo(classes) {
 	);
 }
 
+const Image = () => {
```

Comment:
Rather than storing the component into an `Image` variable, let's just move the static query inside the `Banner` function component. You can place it right next to all the other `const` variables inside the `Banner` component.

It will look something like
```js
function Banner() {
  const data = useStaticQuery(...);
  
  return ... // stuff that makes up the Banner component here
}
```

See my other suggestion below on how to inline the `<Img>` component.